README.md

```diff
@@ -1,39 +1,42 @@
-# vscode-opa
+# vscode-OPA
```

Comment:
Not sure about this. I'd either leave it be (as it reflects the name of the repo) or use "VS Code OPA extension" or something like that?

Reply:
Oops, that's a mistake. Good spot!